.github/workflows/build-main.yml

```
@@ -23,17 +23,19 @@ jobs:
         uses: actions/setup-node@v2
```

actionlint (1.7.9)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)